ppr-ui/src/components/mhrRegistration/YourHome/RebuiltStatus.vue

```diff
@@ -10,8 +10,8 @@
             filled
             id="rebuilt-status-text"
             counter="280"
-            v-model="rebuiltRemarks"
-            :rules="maxLength(280)"
+            v-model.trim="rebuiltRemarks"
+            :rules="customRules(invalidSpaces(), maxLength(280))"
```

Comment:
This actually didn't work that well for me. It would throw an error each time I type space between the words.

Reply:
Good point. Updated
try `validate-on-blur` prop